backend/controller/ConfigController.ts

```diff
@@ -67,19 +68,27 @@ export class ConfigController {
      * Sets a default value for non existing configentries
      */
     public static async ensureConfig(): Promise<void> {
-        const nullInitialized = [
-            ConfigKey.MAX_SEQUENCES_PER_USER,
-            ConfigKey.MAX_SLIDES_PER_SEQUENCE,
-        ];
-        for (const x of nullInitialized) {
-            const entry = await DBConfigEntry.findOneBy({ key: x });
-            if (entry !== null) {
-                continue;
+        const defaultValueMap: Map<unknown, ConfigKey[]> = new Map([
```

Comment:
I feel like this would be more readable if it was just
```js
{
    [ConfigKey.MAX_SEQUENCES_PER_USER]: -1,
    [ConfigKey.MAX_SLIDES_PER_SEQUENCE]: -1,
    [ConfigKey.OPEN_REGISTRATION]: false
}
```